install/install.php

```diff
@@ -434,7 +432,7 @@ function next_form() {
       }
    } else if (!empty($newdatabasename)) { // create new db
       // try to create the DB
-      if ($link->query("CREATE DATABASE IF NOT EXISTS ".AbstractDatabase::quoteName($newdatabasename))) {
+      if ($link->query("CREATE DATABASE IF NOT EXISTS ".DBmysql::quoteName($newdatabasename))) {
```

Comment:
`DBmysql` should not be used directly. Most of the time we can rely on `$DB` but here it is not yet instantiated.